trdg/utils.py

```diff
@@ -145,4 +145,5 @@ def get_text_height(image_font: ImageFont, text: str) -> int:
     """
     Get the width of a string when rendered with a given font
     """
-    return image_font.getsize(text)[1]
+    left, top, right, bottom = image_font.getbbox(text)
+    return bottom - top
```

Comment:
In my local testing with Pillow v10.2, this produces different results than before. Comparing the output of `getsize` and `getbbox` with Pillow 9.5.0 (which has both APIs):

```
trdg -c 5
...
text "a" get_bbox (0, 16, 16, 32) get_size (16, 32)
```

So counter-intuitively, `get_size` was return the value of the `bottom` of the bounding box rather than `bottom - top`.

Reply:
There is a working version of the fix at https://github.com/robertknight/TextRecognitionDataGenerator/commit/5abfc4f79db90014c6c695219458886c83801a6b.

Reply:
Hm, yes, you are right. [Here](https://github.com/tensorflow/models/commit/0aa039f16361f14ee587a9b2d99a16be01d9dcf9) is a similar fix.

The [official Pillow documentation](https://pillow.readthedocs.io/en/stable/deprecations.html) suggests the fix with `bottom - top`, but that would indeed give different results.